.circleci/config.yml

```diff
@@ -6,36 +6,68 @@ commands:
     parameters:
       py_version:
         type: string
+      nox_args:
+        type: string
+        default: ""
     steps:
       - checkout
       - run:
           name: "Preparing environment"
           command: |
-            sudo apt-get update
-            sudo apt-get install -y openjdk-11-jre
-            sudo pip install nox
+            if ! type sudo 2> /dev/null; then
+              apt-get update
+              apt-get install -y openjdk-11-jre
+              pip install nox
+            else
+              sudo apt-get update
+              sudo apt-get install -y openjdk-11-jre
+              sudo pip install nox
+            fi
       - run:
           name: "Testing OmegaConf"
           command: |
             export NOX_PYTHON_VERSIONS=<< parameters.py_version >>
-            nox --add-timestamp
+            nox --add-timestamp << parameters.nox_args >>
```

Comment:
In a pypy conda env, running `nox -s test_omegaconf_pypy` succeeds, but running `NOX_PYTHON_VERSIONS=pypy3.7 nox` fails, since some other sessions (e.g. docs, lint, etc) will not be able to run against the pypy executable. This is why I added `<< parameters.nox_args >>` to the circleci config file.

Reply:
Adding a new session specifically for pypy (something like this) could work:
```python
@nox.session(python=["pypy-3.8"])
def omegaconf_pypy(session):
    deps(session)
    session.run("pytest")
```

Reply:
That's basically what I ended up doing with the `def test_omegaconf_pypy(session): ...` below.

The `parameters.nox_args` here is just to make sure that other sessions besides `test_omegaconf_pypy` don't get run.
Without the `parameters.nox_args`, nox tries to run e.g. the `benchmark` and `coverage` sessions using the `python` executable it finds in `$PATH` (which is aliased to pypy).